File: java_backup/my java/sanhita/Leapyear.java
import java.util.*;
class Leapyear
{
    public static void main(String args[])
    {
        int y;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter year");
        y=sc.nextInt();
        if(y%100==0)
        {
            if((y%4==0)||(y%400==0))
            System.out.println("Leap year");
            else
             System.out.println(" not Leap year");
            }
        }
    }
    
         

File: c_backup/BIN/C Prog/leapyear.c
#include<stdio.h>
int main()
{
	int year;
	printf("Enter the year to be checked: ");
	scanf("%d",&year);
	if(year%4==0 && year%400==0 || year%100!=0)
		printf("The year is a leap year!!.. :-)");
	else
		printf("The year is not a leap year :-(");
	return 0;
}
